_pages/en/strategy-communications.md

```diff
@@ -17,21 +17,21 @@ permalink: /strategy-communications.html
 
 ## Purpose
 
-To elaborate on the re-usable methods the IT Strategy team will have at its disposal in communicating effectively its strategies to various audiences.
+To elaborate iterative process of soliciting the knowledge, experience judgement and values of wide range of different shareholders
```

Comment:
To be clear, this communication strategy is not the iterative part. It is a strategy that provides multiple means to inform, seek contribution, endorse, and approve the iterative strategies.

E.g. Q1 Continuous Learning is one of the many strategies we will be making. It will use this document "team's communication strategy" to build its own communication plan (e.g. go to WMC, go to NUMCC, go to DG Comms, etc...). Based on the Q1 Continuous Learning content which is more HR-related. But if we do another quartely strategy, say "Q2 Agile Teams", we will need to reuse this "Team's communication strategy" to build the Q2 Agile Teams communciations plan (e.g. go to MTLC, to go SLF, to go ARC perhaps)

So the original statement should be kept:
"To elaborate on the re-usable methods the IT Strategy team will have at its disposal in communicating effectively its strategies to various audiences"